From 7d00373015a25d6a9c7c429479cc89f2f5fcd76b Mon Sep 17 00:00:00 2001
From: Noah Davis <noahadvs@gmail.com>
Date: Mon, 1 Mar 2021 15:00:22 -0500
Subject: [PATCH] [RangeSlider] Lower Kirigami version to 2.14

Forgot 2.16 wasn't a released version of Kirigami

(cherry picked from commit 2d431b75b338bb1aae04c38857468928e020823d)
---
 style/qtquickcontrols/RangeSlider.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/style/qtquickcontrols/RangeSlider.qml b/style/qtquickcontrols/RangeSlider.qml
index 3d1b251..479eab0 100644
--- a/style/qtquickcontrols/RangeSlider.qml
+++ b/style/qtquickcontrols/RangeSlider.qml
@@ -7,7 +7,7 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
 import QtQuick.Templates 2.15 as T
-import org.kde.kirigami 2.16 as Kirigami
+import org.kde.kirigami 2.14 as Kirigami
 import "impl"
 
 T.RangeSlider {
-- 
2.25.1

